packages/ava/src/ckb/ckb.ts

```diff
@@ -0,0 +1,80 @@
+import { base } from './base';
```

Comment:
this file should be renamed to `index.ts`

Reply:
got it (the Files changed list makes me confusion
![image](https://user-images.githubusercontent.com/15177000/201531050-fa3c8607-0147-4b65-8d0d-7f135fa1bf1a.png)